stock_available_to_promise_release/models/stock_location_route.py

```diff
@@ -7,6 +7,14 @@
 class Route(models.Model):
     _inherit = "stock.location.route"
 
+    allow_unrelease_on_cancel = fields.Boolean(
```

Comment:
The fact that the action cancel triggers unrelease is not the matter here.
It is the fact that unrelease creates a return of done moves.
```suggestion
    allow_unrelease_return_done_move = fields.Boolean(
```